libdrgn/program.c

```diff
@@ -1108,6 +1108,140 @@ drgn_program_find_symbol_by_address(struct drgn_program *prog, uint64_t address,
 	return NULL;
 }
 
+DEFINE_VECTOR(symbolp_vector, struct drgn_symbol *);
+
+enum {
+	SEARCH_NAME = (1 << 0),
+	SEARCH_ADDRESS = (1 << 1),
+	SEARCH_ALL = (1 << 2),
```

Comment:
It's implied by the context, but just for extra clarity let's prefix these with `SYMBOL_`.